packages/shared/src/utils/constants/idv-options.ts

```diff
@@ -1,3 +1,3 @@
 import { localize } from '@deriv/translations';
 
-export const IDV_NOT_APPLICABLE_OPTION = { id: '#NA', text: localize('I don’t have any of these') };
+export const getIDVNotApplicableOption = () => ({ id: '#NA', text: localize('I want to do this later') });
```

Comment:
are we removing `I don't have any of these` option from the list?

Reply:
There is a card to replace the test. Making the changes in the same PR and adding that card as a sub card

Reply:
Please add that card as sub task